src/pages/editProfile.tsx

```diff
@@ -153,101 +204,107 @@ export default function Home() {
-          <CommonTextField
-            header={editProfileHeader.displayName}
-            value={displayName}
-            handleValueChange={handleDisplayNameChange}
-            char_limit={CHAR_LIMIT.MAX_DISPLAY_NAME}
-            isErr={isPressSubmit && displayNameErr.err}
-            errMsg={displayNameErr.msg}
-          />
-
-          <CommonTextField
-            header={editProfileHeader.description}
-            value={description}
-            handleValueChange={handleDescChange}
-            isMultiLine={true}
-            char_limit={CHAR_LIMIT.MAX_DESCRIPTION}
-            isErr={isPressSubmit && descriptionErr.err}
-            errMsg={descriptionErr.msg}
-          />
-
-          <Box style={{ width: "18vw" }}>
-            <CommonDropdown
-              header={editProfileHeader.gender}
-              value={gender}
-              handleValueChange={handleSelectChange}
-              items={Object.values(Gender)}
-            />
+      <Stack alignItems="center">
+        <Card sx={EditProfileStyle.Card}>
+          <Box display="flex">
+            <Box sx={{ flexGrow: 1 }}></Box>
+            <IconButton onClick={handleOpenModal}>
+              <CloseIcon fontSize="large" color="secondary" />
+            </IconButton>
           </Box>
-        </Box>
-
-        <Button variant="contained" onClick={editProfileBtnOnClick}>
-          SAVE
-        </Button>
-        <Typography variant="body1" color="warning.main">
-          ท่านต้องกด SAVE เพื่อบันทึกข้อมูลใหม่
-        </Typography>
+          <Stack
+            spacing={2}
+            sx={{ marginTop: "-40px" }}
+            alignItems="center"
+            justifyContent="center"
+          >
+            <IconButton
+              onClick={handleCancelImageChip}
+              color="secondary"
+              style={{
+                padding: 0,
+                backgroundColor: "black",
+                position: "relative",
+                top: "70px",
+                left: "80px",
+                zIndex: "1",
+              }}
+            >
+              <CloseIcon color={showImageUploadError.err ? "error" : "primary"} fontSize="medium" />
+            </IconButton>
+
+            <Avatar alt="Anya" sx={EditProfileStyle.Avatar}>
+              {input.image && (
+                <Image
+                  src={input.image}
+                  alt="Upload avatar"
+                  width={200}
+                  height={200}
+                  style={!isImageUpload ? { opacity: "0.5" } : { objectFit: "cover" }}
+                />
+              )}
+              <IconButton
+                sx={EditProfileStyle.DeletePictureButton}
+                aria-label="upload picture"
+                component="label"
+              >
+                <input onChange={handleImageChange} hidden accept="image/*" type="file" />
+                <CameraAltIcon sx={{ fontSize: "100px" }} />
+              </IconButton>
+            </Avatar>
```

Comment:
Image seem a little compress from the actual pic
the first pic is the actual pic and the second is pic in edit profile
<img width="286" alt="image" src="https://user-images.githubusercontent.com/62459518/226860085-3903e1c9-2340-4253-acf8-214916aa57c2.png">
<img width="296" alt="image" src="https://user-images.githubusercontent.com/62459518/226859828-d3b80210-b92c-43cc-9eb1-92b463ddbc4b.png">

Reply:
Solved